server/app.js

```diff
@@ -43,8 +43,20 @@ listener.route('/aria-at/:branch*').get(
     proxyMiddleware.proxyPath(baseUrl)
 );
 
-// Conditionally initialize mock server
-if (process.env.ENVIRONMENT === 'dev') {
+// Conditionally initialize github workflow service, or mock automation scheduler
+if (
+    process.env.ENVIRONMENT === 'production' ||
+    process.env.AUTOMATION_CALLBACK_FQDN
+) {
+    require('./services/GithubWorkflowService')
+        .setup()
+        .catch(error => {
+            console.error(
+                'Failed to initialize github workflow service:',
+                error
+            );
```

Comment:
```suggestion
            error.message = `Failed to initialize github workflow service: ${error.message}`;
            throw error;
```

Or remove the `catch` invocation altogether. I just want the process to quit because this is a critical error.